NickvisionTubeConverter.Shared/Models/VideoUrlInfo.cs

```diff
@@ -124,14 +124,21 @@ private VideoUrlInfo(string url)
                     dynamic ytdlp = Python.Runtime.Py.Import("yt_dlp");
                     var ytOpt = new Dictionary<string, dynamic>() {
                         { "quiet", true },
-                        { "merge_output_format", "/" }
+                        { "merge_output_format", "/" },
+                        { "ignoreerrors", true }
                     };
                     Python.Runtime.PyDict videoInfo = ytdlp.YoutubeDL(ytOpt).extract_info(url, download: false);
```

Comment:
PyObject is a subclass of DynamicObject, so it worked without "?", but for style consistency I added it

Reply:
yeah it's legal without the `?`, the compiler just throws a warning. But for "modern C# style" it's better to put `?`. When using `var` and checking `if == null` the `var` type will automatically add the `?`, but because we are defining the type manually, we must manually put `?`.